docusaurus/docs/publish-a-plugin/provide-provisioning.md

```diff
@@ -0,0 +1,60 @@
+---
+id: provide-provisioning
+title: Provision your plugin
+description: How to add provisioning to your plugin to potentially speed up your plugin review process.
+keywords:
+  - github
+  - provisioning
+  - provision
+  - review
+sidebar_position: 5
+---
+
+# Plugin provisioning
+
+Developers of community plugins often ask us these questions:
+
+- How long does it take to review plugins for inclusion in the Grafana [plugin catalog](https://grafana.com/plugins)?
+- How can Grafana Labs speed up the time it takes us to review their submissions?
+
+We have heard you. Our response to these questions has been to allow plugin developers themselves to speed the review process by creating a test environment.
+
+The mechanism for including such an environment with your plugin submission is called _provisioning_. Provisioning refers to the process of preparing and configuring your plugin environment to meet specific requirements, making it easier for reviewers to assess your plugin comprehensively.
+
+## Mechanism to provide for provisioning
+
+Grafana can be configured to have resources installed and enabled through a mechanism known as provisioning, where resources are configured in a YAML file under a `/provisioning` directory.
+
+We have done this for some time in the [create-plugin tool](../get-started/get-started.mdx) for app plugins so that they can be automatically enabled (that is, provisioned) when using our Docker development environment.
```

Comment:
The reference to past/ "we're now" makes this read more like a blog/point-in-time article. Suggest removing and just saying describing what the tool does by default and the new command. Optionally can include a version number such as "Since `create-plugin` v2.7.0, we generate..."

Reply:
done.